api/api.py

```diff
@@ -423,6 +423,10 @@ def read_json(path: str, default: str = None) -> StructuredDataType:
     default: If not `None` and the file at `path` does not exist, this value will be returned."""
   pass
 
+def decode_yaml(yaml: str) -> StructuredDataType:
```

Comment:
This is a pre-existing problem but I'm unfamiliar with `StructuredDataType`, is that a thing a pythonista would be familiar with? Should we explain it somehow?

Reply:
AFACT, the python api docgen expands this to 

Return type:	Union [ Dict [ str , Any ], List [ Any ]]
https://docs.tilt.dev/api.html#api.decode_json